CONTENTS.md

```diff
@@ -0,0 +1,423 @@
+# Contents
+
+This document describes in detail the contents of this repository. It is still a *work in progress*,
+but do not hesitate to report if something is missing.
+
+## Overview
+
+The repository contains a set of tools to aid in the process of building, testing and releasing
+(open)SUSE based distributions. The [Tools](#tools) section enumerates and describes all these
+tools, including links to documentation, source code and some information about where they are used.
+
+Apart from these tools, the repository includes:
+
+* Some documenation in the [docs](docs) directory.
+* A Python module called [osclib](osclib) which includes code that is shared by several tools. They
+  are available in the `osclib` package.
+* A Docker-based [tests suite](tests). The Docker manifests and the Docker Compose files are
+  located in the [dist](dist) directory.
+* [GoCD](https://www.gocd.org) configuration files in [gocd](gocd). GoCD is an open source CI/CD
+  server that it is used to deploy the bots on OBS.
+* A set of [Tampermonkey](https://www.tampermonkey.net) scripts (see [userscript](userscript)
+  directory) to extend OBS features when using the web interface.
+* Several [systemd](systemd) units, as many of the tools (e.g., the `staging-bot`) make use of
```

Comment:
A couple of days ago the review bot units were removed, these are the ones left

```suggestion
* Several [systemd](systemd) units: the Metrics and OBS Operator tools make use of
```

Reply:
The systemd units are gone - we run the bots in gocd since 2019